source/Octopus.Cli/Commands/Releases/CreateReleaseCommand.cs

```diff
@@ -198,7 +201,7 @@ public async Task Request()
             }
         }
 
-        async Task<ReleasePlan> BuildReleasePlan(ProjectResource project)
```

Comment:
This project being passed around is unecessary since it is set as a field in the entry method.